pandas/core/generic.py

```diff
@@ -7784,25 +7784,41 @@ def _doc_parms(cls):
 %(outname)s : %(name1)s or %(name2)s (if level specified)\n"""
 
 _bool_doc = """
-
 %(desc)s
 
+Returns True if one (or more) elements are non-zero,
+not-empty or not-False.
+
 Parameters
 ----------
 axis : %(axis_descr)s
 skipna : boolean, default True
     Exclude NA/null values. If an entire row/column is NA, the result
-    will be NA
+    will be NA.
 level : int or level name, default None
     If the axis is a MultiIndex (hierarchical), count along a
-    particular level, collapsing into a %(name1)s
+    particular level, collapsing into a %(name1)s.
 bool_only : boolean, default None
     Include only boolean columns. If None, will attempt to use everything,
     then use only boolean data. Not implemented for Series.
+**kwargs : Additional keywords have no effect but might be accepted for
```

Comment:
@datapythonista is this the language generally recommending here?

Reply:
I didn't see this earlier. I think the description is similar to the ones used in `kwargs` with similar behavior. But it should go in the next line, as in the other parameters, and in this line simply leave `**kwargs`